src/lcancel.c

```diff
@@ -196,35 +197,53 @@ void LCancel_Think(LCancelData *event_data, FighterData *hmn_data)
     JOBJ *hud_jobj = event_data->hud.gobj->hsd_object;
 
     bool should_update_fastfall_hud = false;
+    bool can_fastfall = hmn_data->phys.self_vel.Y < 0;
 
     // log fastfall frame
     // if im in a fastfall-able state
     int state = hmn_data->state_id;
     //if ((state == ASID_JUMPF) || (state == ASID_JUMPB) || (state == ASID_JUMPAERIALF) || (state == ASID_JUMPAERIALB) || (state == ASID_FALL) || (state == ASID_FALLAERIAL) || ((state >= ASID_ATTACKAIRN) && (state <= ASID_ATTACKAIRLW))
```

Comment:
Go ahead and delete stale code when you come across it and it's no longer needed.